assets/src/containers/SideDrawer.js

```diff
@@ -30,19 +30,25 @@ function SideDrawer (props) {
     classes,
     toggleDrawer,
     courseId,
-    courseInfo
+    courseInfo,
+    enrollmentType,
+    isAdmin
   } = props
 
   const [selectedIndex, setSelectedIndex] = useState(false)
 
+  const isTeacherOrAdmin = () => {
+    return isAdmin || enrollmentType === 'TeacherEnrollment'
+  }
+
```

Comment:
Can we use the utility function you've defined here?

Reply:
Yeah I missed that one, thanks.